devices/sonoff/snzb-03p_motion_sensor.json

```diff
@@ -0,0 +1,192 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "eWeLink",
+  "modelid": "SNZB-03P",
+  "vendor": "SONOFF",
+  "product": "SNZB-03P",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0107",
+        "endpoint": "0x01",
+        "in": [
+          "0x0000",
+          "0x0001",
+          "0x0406",
+          "0x0500",
+          "0xFC11"
+        ]
+      },
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "awake": true,
+          "refresh.interval": 7265,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "eval": "Item.val = Attr.val / 2;",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "config/delay",
+          "refresh.interval": 3600,
```

Comment:
You might want to doublecheck the refresh interval, as reporting is significantly lower

Reply:
In a perfect world, "refresh.interval" is never used, it's just a security for me, the setting is only changed 2 times in a year.
Have just copy/paste other DDF (working I hope)

Reply:
Yeah, absolutely agree. I guess it doesn't matter anyway if 1-2 reports get lost, the poll is very unlikely to happen at all.